_posts/2017-01-13-create-android-background-service.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title: Creating background services in Android
+ghavatar: 2993240
+ghname: jarlefosen
+jobtitle: Platform Engineer
+tags: [android, sdk]
+---
+
+## Introduction
+
```

Comment:
We should really mask this posts as "research" for an upgraded version of our "welcome song". And link to one of Torbjørns posts. Agree?